### PR TITLE
Port the application to rails 4.2.4

### PR DESCRIPTION
**Pull request by jforberg**
Also clean out some gunk which was probably not necessary.

### CODE REVIEW

**Comment by davidwessman** [comment]
Should we also start updating some gems?

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1,32 +1,17 @@
...
- * Any CSS and SCSS file within this directory, lib/assets/stylesheets, vendor/assets/stylesheets,
- * or vendor/assets/stylesheets of plugins, if any, can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the top of the
- * compiled file, but it's generally better to create a new file per style scope.
- */
-
 /* Override for Bootstrap 3 font location */
 /* Start importing Bootstrap */
-@import "bootstrap-sprockets";
```
Comment (davidwessman) [inline app/assets/stylesheets/application.scss]: https://github.com/twbs/bootstrap-sass#a-ruby-on-rails

Sprockets should be loaded first ;)

**Comment by davidwessman** [comment]
LGTM, I made the same changes on a local branch to get it going ;) 

Did you update gems as well?

File: app/assets/stylesheets/application.scss (hunk 1 above)
Comment (jforberg) [inline app/assets/stylesheets/application.scss]: Bah, sprockets sass integration is so buggy. I swapped these lines while debugging another error that turned out to be caused by some font-url calls. For the record, font-url is completely broken and should be avoided in favour of asset-url.

Will swap the lines back.

**Comment by jforberg** [comment]
All gems are at the latest versions except mysql2 which is held back because the latest version breaks activerecord.

I also removed the version locks for the other packages. In general packages should not be held back unless there is a reason to do so, described in a comment or such.

**Comment by davidwessman** [comment]
This just needs a rebase and a merge?

**Comment by jforberg** [comment]
Yes but some downtime is likely. Someone should probably babysit the deploy.

Feel free to do the merge if you have time. You're currently the only one who knows how the deploy system works, I think.

**Comment by davidwessman** [comment]
We can try it out on stage.

**Comment by davidwessman** [comment]
Fixed in #235 and #237

**Pull request closed without merging**